CMakeLists.txt

```diff
@@ -3,6 +3,12 @@
 cmake_minimum_required(VERSION 3.13)
 enable_testing()
 
+find_program(CCACHE ccache)
+if(CCACHE)
+    set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
+    set(CMAKE_C_COMPILER_LAUNCHER ${CCACHE})
+endif()
```

Comment:
This seems like it should be put under an option, so it's not forced to be used just because ccache exists on the system.

Reply:
I can do it.